### PR TITLE
rename category --> family (fix #288)

### LINKED ISSUE #288: rename "category" --> "family"
Per the description in [_data/meta.yml](../blob/gh-pages/_data/meta.yml). Also, it's more intuitive what it means, IMO.

### PR DESCRIPTION
**Pull request by waldyrious**
Also use "ruletype" instead of "category" in app.coffee, to make it more explicit.

### CODE REVIEW

**Comment by waldyrious** [comment]
Hope this is ok :)

File: assets/js/app.coffee [hunk 1]
```diff
@@ -29,8 +29,8 @@ class Choosealicense
     my: "top center"
     at: "bottom center"
 
-  # Annotation categories as defined in `_config.yml`
-  categories:
+  # Annotation families as defined in `_config.yml`
+  families:
```
Comment (waldyrious) [inline assets/js/app.coffee]: oops, this shouldn't have been changed. Will revert in a moment.

**Comment by bkeepers** [comment]
This seems fine to me.

@benbalter?

**Comment by benbalter** [comment]
Makes sense. Thanks @waldyrious! :rabbit: :kimono: :ice_cream: :necktie:

**Pull request merged**